src/Stock.js

```diff
@@ -90,7 +134,7 @@ class Stock extends React.Component {
                 <StockForm sendStockFormData={this.receiveStockFormData}></StockForm>
 
                 {this.state.stockTimeValues.length > 0 ?
-                    <Plot
+                    <><Plot
```

Comment:
Let's put the extra <> on a new line too. Just nice to see the hierarchy visually with indentation (and when you do that, some of the indentation might be off, but if you do Ctrl+K, Ctrl+F, it'll fix it for you).

Reply:
Decided I would just do this change (as I already had it done while checking to see if it would help the flow of the code)